docs/README.md

```diff
@@ -1,3 +1,97 @@
-# About
+# Spiderpool
 
-spiderpool
+## Introduction
+
+The Spiderpool is an IP Address Management (IPAM) CNI plugin that assigns IP addresses for kubernetes clusters.
+
+Currently, it is under developing stage, not ready for production environment yet.
+
+Any Container Network Interface (CNI) plugin supporting third-party IPAM plugins can use the Spiderpool,
+such as [MacVLAN CNI](https://github.com/containernetworking/plugins/tree/main/plugins/main/macvlan),
+[VLAN CNI](https://github.com/containernetworking/plugins/tree/main/plugins/main/vlan), [IPVLAN CNI](https://github.com/containernetworking/plugins/tree/main/plugins/main/ipvlan) etc.
+The Spiderpool also supports
+[Multus CNI](https://github.com/k8snetworkplumbingwg/multus-cni)
+case to assign IP for multiple interfaces.
+More CNIs will be tested for integration with Spiderpool.
+
+## Why Spiderpool
+
+Most overlay CNIs, like
+[Cilium](https://github.com/cilium/cilium)
+and [Calico](https://github.com/projectcalico/calico),
+have a good implementation of IPAM, so the Spiderpool is not intentionally designed for these cases, but maybe integrated with them.
+
+The Spiderpool is intentionally designed to use with underlay network, where administrators can accurately manage each IP.
+
+Currently, in the community, the IPAM plugins such as [whereabout](https://github.com/k8snetworkplumbingwg/whereabouts), [kube-ipam](https://github.com/cloudnativer/kube-ipam),
+[static](https://github.com/containernetworking/plugins/tree/main/plugins/ipam/static),
+[dhcp](https://github.com/containernetworking/plugins/tree/main/plugins/ipam/dhcp), and [host-local](https://github.com/containernetworking/plugins/tree/main/plugins/ipam/host-local),
+few of them could help solve complex underlay-network issues, so we decide to develop the Spiderpool.
+
+BTW, there are also some CNI plugins that could work on the underlay mode, such as [kube-ovn](https://github.com/kubeovn/kube-ovn) and [coil](https://github.com/cybozu-go/coil).
+But the Spiderpool provides lots of different features, you could see [Features](#features) for details.
+
+## Features
+
+The Spiderpool provides a large number of different features as follows.
+
+* Based on CRD storage, all operation could be done with kubernetes API-server.
+
+* Support for assigning IP addresses with three options: IPv4-only, IPv6-only, and dual-stack.
+
+* Support for working on the clusters with three options: IPv4-only, IPv6-only, and dual-stack.
+
+* Support for creating multiple ippools.
+  Different namespaces and applications could monopolize or share an ippool.
+
+* An application could specify multiple backup ippool resources, in case that IP addresses in an ippool are out of use. Therefore, you neither need to scale up the IP resources in a fixed ippool, nor need to modify the application yaml to change a ippool.
```

Comment:
```suggestion
* An application could specify multiple backup ippool resources in case IP addresses in an ippool are out of use. Therefore, you neither need to scale up the IP resources in a fixed ippool, nor need to modify the application yaml to change an ippool.
```